### DISCUSSION TITLE
Documentation not showing when using FastEndpoints

### DISCUSSION THREAD

**Discussion by grosch-intl** [scalar/scalar#3872] [Q&A] [answered]
When I go to the scalar/v1 endpoint, I see my DTO, which includes XML doc comment summary tags on the class and both public properties.  However, the comments aren't there.  I've tried with and without the `GenerateDocumentationFile` property in the csproj file.

**Comment by hanspagel** [comment]
Any chance you can share your OpenAPI document with us (/scalar/v1)?

**Comment by xC0dex** [comment, marked as answer]
Hey @grosch-intl, which OpenAPI document generator are you using? The built-in generator (shipped with .NET 9) doesn't support XML comments yet.

**Reply by grosch-intl** [reply]
That makes it kinda useless 😢 Is there a github issue where you saw that so I can add a watch to it?

**Reply by xC0dex** [reply]
Hey @grosch-intl, you can subscribe this issue: https://github.com/dotnet/aspnetcore/issues/39927! You can still use `Swashbuckle` or `NSwag` for the OpenApi generation with .NET 9!

**Reply by xC0dex** [reply]
Hey @grosch-intl,

There is also a third-party library that adds XML comments support to `Microsoft.AspNetCore.OpenApi`: https://github.com/martincostello/openapi-extensions

**Reply by grosch-intl** [reply]
@xC0dex Thanks.  I added that and it works well with the model classes. Still struggling to get documentation on the endpoints themselves, but as it doesn't appear in the generated json, I'll post a message in the extensions repo.
